### ISSUE TITLE
"New Instance" menu would make more sense than "Vanilla Packs"

### ISSUE THREAD

**Issue by Obscure2020** [ATLauncher/ATLauncher#563]
### (There is an audio recording of this post located at the bottom, if you'd like to listen to me read this post, for your convenience.)

**Is your feature request related to a problem? Please describe.**
I recently had a usability issue with ATLauncher. I eventually figured out how to do what I wanted to do, but I realized I might as well tell you what happened and suggest a solution.

You see, I've been using ATLauncher for a month now, and I've been pretty pleased with its features. I use a pretty large stack of Fabric mods to play Minecraft, most of them being performance enhancers and optimizers, and the rest being quality-of-life improvements. I was previously using GDLauncher, but I disliked the inconsistent behavior GDLauncher displayed regarding Alpha vs Beta vs Full-Release versions of mods when new versions were released by their devs. I much prefer ATLauncher's solution of allowing me to select exactly what version I'd like when new versions of mods are released, as well as allowing me to download mods from both CurseForge _and_ Modrinth. A+, guys. I love it.

The usability issue arose within the last few days, when I went to create a new instance. You see, I encountered a bug in one of the mods I use, and for the purpose of testing that mod and filing a bug report on it, I went to create a new Fabric instance. And that's where I hit a stumbling block.

It had been almost a month since I started using ATLauncher, and for the life of me, I _could not remember_ how to make a new instance! I was looking around going "Where's the `New Instance` button?? Shouldn't it be in the `Instances` menu?? Why is there no `New Instance` button??"

Eventually I re-figured-out that you needed to go into the `Vanilla Packs` menu to create a new instance. Since this incident occurred, I've come to believe that this wasn't just a bout of forgetfulness, this was the result of flawed design. I'd like to suggest a new way of organizing ATLauncher to make it more obvious how to create new instances and servers.

**Describe the solution you'd like**
1. Remove the `Vanilla Packs` menu entirely. Erase it from the sidebar.

2. Add a `New Instance` or `Create New Instance` button to the `Instances` menu, and have that button bring up the version and loader selector (the one currently located in `Vanilla Packs`) in a new _sub-window_, but with only the `Create Instance` button at the bottom of the menu.

3. Add a `New Server` or `Create New Server` button to the `Servers` menu, and have that button bring up the version and loader selector in a new sub-window, but with only the `Create Server` button at the bottom of the menu.

**Describe alternatives you've considered**
I'd really prefer the solution listed above, but if you don't feel that my solution is worth implementing, or if you'd like to implement a _temporary_ improvement while you work on the solution I've suggested above, you could do this:
- Rename the `Vanilla Packs` menu to `New Instances`.

https://user-images.githubusercontent.com/69320464/161441500-86fc2ceb-ed36-4c89-8699-fa37ba30cea6.mp4

**Comment by RyanTheAllmighty** [comment]
Yeah I hate the name of that tab honestly. There used to be actual packs setup in the launcher that were Vanilla, Vanilla + Fabric and Vanilla + Forge so they were actually, at one point in time, Vanilla Packs. When I switched to the current format of that tab maybe 9ish months ago it just stayed the same, purely for convenience for people who were used to finding that there.

I think the solution here will be to keep the tab and also add those buttons, as it makes sense that you would look in the Instances/Servers tab to make a new instance, so I think it's probably a good middle ground to have both

**Comment by Obscure2020** [comment]
Yeah, that sounds reasonable. Having the new buttons for new users and keeping the old `Vanilla Packs` menu for those familiar with it could be useful.

The only thing I would say is that I don't think redundancy is always helpful. I agree that we should cater to older users, but I agree more with "it makes sense that you would look in the Instances/Servers tab to make a new instance" (or server). I think if we can get the new buttons working nicely and if people like them, we should consider removing the `Vanilla Packs` menu at some point in the future.

Might there be some way to let users vote on this? I think it could be helpful, after the addition of the new buttons, to poll the userbase and ask whether they prefer the buttons or the `Vanilla Packs` menu, and judge how worthwhile it is to keep the old menu around based on the results of the poll.

**Comment by RyanTheAllmighty** [comment]
Honestly I don't have too big of an issue keeping the old one around. It will likely reuse the same code for the tab in a dialog anyway, but yeah will see how it goes, there will be tracking on it anyway, so will be able to see if the tab is used much after the change, and I can likely make it configurable and do some A/B testing on it if I really feel the need to go that far (probably not that far though honestly)

**Comment by Obscure2020** [comment]
Personally I think you might be surprised by how much users will appreciate a more _intuitive_ solution, but who knows. I could be wrong.

**Comment by Doomsdayrs** [comment]
#806 has completed this?

**Comment by Obscure2020** [comment]
I'd say #806 is a start. It's still not great.

The name is still wrong. It's called the `Create Packs` menu but you don't use it to create packs. You use that menu to create Instances and Servers.

My original proposed solution is still a superior idea. The `Create Packs` menu should not be listed on the sidebar at all. The functionality of creating new Instances and Servers should be moved into the `Instances` and `Servers` menus.

**Comment by Doomsdayrs** [comment]
> My original proposed solution is still a superior idea. The `Create Packs` menu should not be listed on the sidebar at all. The functionality of creating new Instances and Servers should be moved into the `Instances` and `Servers` menus.

#807 also adds support for this, redirecting the user when there is no instances / servers

**Comment by Obscure2020** [comment]
This issue should remain open until further notice. I'll continue to keep an eye on this. It's possible I may end up creating a PR myself to demonstrate my proposed behavior.
